test/Market.t.sol

```diff
@@ -1580,10 +1583,15 @@ contract Market_convertPurchasingTokenDecimalsToRemovalDecimals is
 
 contract Market_calculates_prices_using_decimal is UpgradeableUSDCMarket {
   function test() external {
-    // expectRevert(_market.calculateCheckoutTotal(1), 0);
```

Comment:
Why is this commented out?

Reply:
doesnt seem like it was ever valid anyways